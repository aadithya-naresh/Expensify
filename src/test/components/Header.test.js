import {shallow} from 'enzyme'
import React from 'react'
import Header from '../../components/Header'
import toJSON from 'enzyme-to-json'
test('Should render the header correctly', () =>{

    const wrapper = shallow(<Header />)
    expect(toJSON(wrapper)).toMatchSnapshot()
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)

    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})


