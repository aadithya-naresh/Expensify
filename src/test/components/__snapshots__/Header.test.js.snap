// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the header correctly 1`] = `
<header>
  <h1>
    Expensify app
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    Help
  </[object Object]>
</header>
`;
